.travis.yml

```diff
@@ -1,4 +1,7 @@
 dist: xenial  # Ubuntu 16.04
+language: node_js
+node_js:
+  - 13
```

Comment:
Why lock to a particular Node version?

Reply:
This matched what was on my workstation, but if there's no good reason to lock it let's remove it.

Reply:
> [Node version 0.10.48 is not supported, please use Node.js 4.0 or higher.](https://travis-ci.org/google/postcss-rename/builds/645738824?utm_source=github_status&utm_medium=notification)

There's our reason 😅

Reply:
Yep there is, thanks for pointing that out.